linker/Linker/Driver.cs

```diff
@@ -362,6 +368,7 @@ static void Usage (string msg)
 			Console.WriteLine ("   --reduced-tracing   Reduces dependency output related to assemblies that will not be modified");
 			Console.WriteLine ("   --used-attrs-only   Attributes on types, methods, etc will be removed if the attribute type is not used");
 			Console.WriteLine ("   --strip-security    In linked assemblies, attributes on assemblies, types, and methods related to security will be removed");
+			Console.WriteLine ("   --remove-resources  Remove link xml resources that were processed (true or false), default to true");
```

Comment:
It'd make sense to call it `--strip-resources` to keep in line with previous options and to make the true default more meaningful